x-pack/plugins/serverless_search/public/application/components/index_management/connector_setup_prompt.tsx

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import {
+  EuiEmptyPrompt,
+  EuiIcon,
+  EuiFlexItem,
+  EuiFlexGroup,
+  EuiButton,
+  EuiPanel,
+  EuiTitle,
+} from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { Connector } from '@kbn/search-connectors';
+
+import { useKibanaServices } from '../../hooks/use_kibana';
+import { useAssetBasePath } from '../../hooks/use_asset_base_path';
+import { useConnectorTypes } from '../../hooks/api/use_connector_types';
+
+interface ConnectorSetupEmptyPromptProps {
+  indexName: string;
+  connector: Connector;
+}
+
+export const ConnectorSetupEmptyPrompt = ({ connector }: ConnectorSetupEmptyPromptProps) => {
+  const { http } = useKibanaServices();
+  const assetBasePath = useAssetBasePath();
+  const { data: connectorTypes } = useConnectorTypes();
+
+  const connectorsIconPath = assetBasePath + '/connectors.svg';
+  const connectorPath = http.basePath.prepend(`/app/connectors/${connector.id}`);
+  const connectorType = connectorTypes?.connectors?.find(
+    (cType) => cType.serviceType === connector.service_type
+  );
+  return (
+    <EuiPanel>
+      <EuiEmptyPrompt
+        icon={<EuiIcon size="xxl" type={connectorsIconPath} />}
+        title={
+          <EuiTitle size="xs">
+            <h5>
+              <FormattedMessage
+                id="xpack.serverlessSearch.indexManagement.indexDetails.overview.emptyPrompt.connector.title"
+                defaultMessage="Set up your connector"
+              />
+            </h5>
+          </EuiTitle>
+        }
+        body={
+          <>
+            <p>
+              <FormattedMessage
+                id="xpack.serverlessSearch.indexManagement.indexDetails.overview.emptyPrompt.connector.body"
+                defaultMessage="This index is managed by an existing connector."
```

Comment:
@ leemthompo 👀  copy here as I modified it slightly from design to read better IMO, but open to improvements/complete re-write as well.

Reply:
This one is perfect 👌